### DISCUSSION TITLE
Nuxt3 compatible tooltip?

### DISCUSSION THREAD

**Discussion by telisjok** [nuxt/nuxt#15959] [Questions]
Is there a tooltip package compatible with Nuxt 3? `v-tooltip` is currently compatible with Nuxt2.

**Comment by wobsoriano** [comment]
Haven't tried but this appeared when I googled

https://floating-vue.starpad.dev/guide/installation.html#nuxt-3

**Comment by michal-stachura** [comment]
Floating Vue will work but currently if you install with their documentation probably you will get an error. Working solution:

1. `yarn add floating-vue`
2. Create file `plugins/floating-vue.js` with below code:
```
import FloatingVue from 'floating-vue'
import 'floating-vue/dist/style.css'

export default defineNuxtPlugin(nuxtApp => {
  nuxtApp.vueApp.use(FloatingVue)
})
```

And you'll be able to use it with latest Nuxt 3

**Reply by agualis** [reply]
Hey @michal-stachura, thanks for the answer. 

Could you share a repo where your solution is working?

I tried the last `2.0.0-beta.7` with `nuxt 3.0.0` but it is not working. 

Also tried adding  'floating-vue/nuxt' to `modules` but that breaks the app.

**Reply by michal-stachura** [reply]
Hey @agualis no problem. Repository here: https://github.com/michal-stachura/examples-nuxt3-tooltip 

Enjoy! :)

**Reply by agualis** [reply]
Oh yeah! So fast! Thank you! 💚
